src/web/js/trove/chart-lib.js

```diff
@@ -1302,10 +1310,14 @@
         });
         // If we have our own image, make the point small and transparent
         if (i < scatters.length) {
+          if (get(p, 'dot-chart')) {
+            // console.log('dot chart scatter plot found');
+            dotChartP = true;
+          }
           $.extend(seriesOptions, {
-            pointSize: hasImage ? 1 : toFixnum(get(p, 'point-size')),
+            pointSize: (hasImage || dotChartP) ? 0.1 : toFixnum(get(p, 'point-size')),
```

Comment:
Rather than doing `A || B` everywhere, I'd recommend setting another predicate that's basically `replaceDefaultSVG`.

Reply:
Done